package.json

```diff
@@ -3,6 +3,11 @@
   "version": "1.0.1",
   "description": "Create components whose prop changes map to a global side effect",
   "main": "index.js",
+  "scripts": {
+    "test": "NODE_ENV=test mocha",
```

Comment:
Can we avoid NODE_ENV? This script will blow up on Windows, e.g. see https://github.com/rackt/redux/pull/379/files. I don't think test runner really cares about it.